@xen-orchestra/lite/src/components/form/FormInput.vue

```diff
@@ -113,12 +131,13 @@ defineExpose({
 
 <style lang="postcss" scoped>
 .form-input,
-.form-select {
+.form-select,
+.form-textarea {
   display: inline-grid;
   align-items: stretch;
 
-  --before-width: v-bind('beforeWidth ?? "1.75em"');
-  --after-width: v-bind('afterWidth ?? "1.625em"');
+  --before-width: v-bind('beforeWidth || "1.75em"');
+  --after-width: v-bind('afterWidth || "1.625em"');
```

Comment:
Since the `beforeWidth` and `afterWidth` props have a `string` type, it's technically possible to provide an empty string (which would then not be `undefined`). This wouldn't be a problem when passing arbitrary data, but when used dynamically, it could happen.

So using `||` instead of `??` handle both cases.